wooey/tests/scripts/without_args.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+import sys
+import os
+
+parser = argparse.ArgumentParser(description="Just do it without arguments!")
+
+def django_setup():
```

Comment:
Yeah, this stuff should be minimal. Just a parser, an added argument, and stuff in main()